code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -1053,20 +1041,6 @@ func (ssc *StorageSmartContract) GetBlocksHandler(_ context.Context, params url.
 }
 
 func (ssc *StorageSmartContract) GetTotalData(_ context.Context, balances cstate.StateContextI) (int64, error) {
-	if ssc != nil {
-		storageNodes, err := ssc.getBlobbersList(balances)
-		if err != nil {
-			return 0, fmt.Errorf("error from getBlobbersList in GetTotalData: %v", err)
-		}
-
-		var totalSavedData int64
-		for _, sn := range storageNodes.Nodes {
-			totalSavedData += sn.SavedData
-		}
-
-		return totalSavedData, nil
-	}
-
 	return 0, fmt.Errorf("storageSmartContract is nil")
```

Comment:
This orphaned error message does not make sense. Maybe something like "total_saved_data is not implemented yet".